src/logic.js

```diff
@@ -162,12 +145,6 @@ define([
                     replacePathInfo(this._xpathParser.parse(to), path);
                 }
             }
-            for (i = 0; i < hashtags.length; i++) {
-                path = hashtags[i];
-                if (path.toHashtag() === from) {
```

Comment:
No code fills me with more regret than when I see `toHashtag` in vellum code. Thanks for hiding some of it :)